### ISSUE TITLE
Port forwarding not working when service port and service targetPort are not the same

### ISSUE THREAD

**Issue by pallxk** [aptakube/aptakube#175]
### Current and expected behavior

**Current behaviour:**

When a service specifies A as the port of the service (1234 in the example below), while targeting port B of matched pods (5678 in the example), the port forwarding of Aptakube will target port A of matched ports, leading to invalid port forwarding.

**Expected behaviour:**

When a service specifies A as the port of the service, while targeting port B of matched pods, the port forwarding of Aptakube should target port B of matched ports.

**Example spec:**

```yaml
spec:
  ports:
    - name: http
      protocol: TCP
      port: 1234            # A
      targetPort: 5678      # B
```

### Environment & Operating System

macOS Sonoma
Kubernetes v1.27.4+orb1

**Comment by goenning** [comment]
Thanks @pallxk 

I suppose if you try to port forward to a specific pod behind that service it works normally?

**Comment by pallxk** [comment]
Yes. If I port forward directly to a specific pod, it works.

**Comment by goenning** [comment]
hey @pallxk this should be fixed now on 1.6.0, but please reopen if it's not! Thanks
